backend/doc/apache.md

```diff
@@ -0,0 +1,93 @@
+# Running Tracim through Apache #
+
+### Installation ###
+
+Install `tracim` first.
+Install `Apache` server and uWSGI:
+
+    sudo apt install apache2 libapache2-mod-wsgi-py3 uwsgi uwsgi-plugin-python3
+
+### Configuration ###
+
+Create a file named `/etc/apache2/sites-available/tracim.conf` containing: 
+
+    <VirtualHost *:80>
+        ServerName tracim
```

Comment:
replace tracim by [server_name] (and explain that user need to change it, tracim is domain_name).

Reply:
done